### PR TITLE
[qca-ecm.init] Add if condition in start function

### PR DESCRIPTION
**Pull request by XiaoliChan**
Without this, the "net.bridge.bridge" will append into /etc/sysctl.d/qca-nss-ecm.conf in every system reboot/startup.

### CODE REVIEW

**Comment by robimarko** [comment]
Wouldnt it be enough to just check whether  `/etc/sysctl.d/qca-nss-ecm.conf` exists?

Also, SoB is missing

**Comment by XiaoliChan** [comment]
> Wouldnt it be enough to just check whether `/etc/sysctl.d/qca-nss-ecm.conf` exists?

Do you mean it needs to check all the file content in `/etc/sysctld.d` ?

**Comment by robimarko** [comment]
No, I mean if the file exists, cause why only check for part of the file if it appends on each boot?

**Comment by XiaoliChan** [comment]
> No, I mean if the file exists, cause why only check for part of the file if it appends on each boot?

Well, I think if the router reset, then only append these lines on the first boot.

**Comment by XiaoliChan** [comment]
I should combine the grep condition XD

**Comment by robimarko** [comment]
Let me rephrase, does `/etc/sysctl.d/qca-nss-ecm.conf` exist after a reboot and what is its content?

**Comment by XiaoliChan** [comment]
> 

In original script, you will see this content after the reboot in [/etc/sysctl.d/qca-nss-ecm.conf]
```
net.netfilter.nf_conntrack_tcp_no_window_check=1  
net.bridge.bridge-nf-call-ip6tables=1  
net.bridge.bridge-nf-call-iptables=1  
net.bridge.bridge-nf-call-ip6tables=1  
net.bridge.bridge-nf-call-iptables=1  
net.bridge.bridge-nf-call-ip6tables=1  
net.bridge.bridge-nf-call-iptables=1  
net.bridge.bridge-nf-call-ip6tables=1  
net.bridge.bridge-nf-call-iptables=1  
...
````

**Comment by robimarko** [comment]
Ok, then just add your Signed-off-by to the patch and I will merge it

**Comment by XiaoliChan** [comment]
Okay, I append the signed information. :)
